tests/zfs-tests/tests/perf/perf.shlib

```diff
@@ -105,7 +105,7 @@ function do_fio_run_impl
 	# the default filesystem (e.g. against a clone).
 	#
 	export DIRECTORY=$(get_directory)
-	log_note "DIRECTORY: " $DIRECTORY
+	log_note "DIRECTORY: " "$DIRECTORY"
```

Comment:
I don't think `log_note` cares about number of arguments, but if we want to wrap the variable into quotes, I don't think we need to close and reopen them before it.

Reply:
You're right, log_note does not care about arguments so changing the behavior here from passing two parameters to one string will not matter.